packages/styles/settings-tools/_s.file-uploader.scss

```diff
@@ -0,0 +1,2 @@
+$fileuploader-border-radius: get-border-radius('m');
```

Comment:
Why don't you use the button variable ?

You decided to use all the buttons mixins and scss token map  but not the border radius and the border width

Reply:
Well done, it's a mistake :)